app/index.js

```diff
@@ -15,7 +15,7 @@ var projectTypes = [{
 var props;
 
 module.exports = generators.Base.extend({
-  prompting: function() {
+  prompting: function () {
```

Comment:
This commit adds some unnecessary whitespace. Please leave the formatting intact.